packages/core/src/components/ui/ProductDetails/ProductDetailsSettings.tsx

```diff
@@ -110,13 +127,26 @@ function ProductDetailsSettings({
     <>
       {!outOfStock && (
         <section data-fs-product-details-values>
-          <ProductPrice.Component
-            data-fs-product-details-prices
-            value={price}
-            listPrice={listPrice}
-            formatter={useFormattedPrice}
-            {...ProductPrice.props}
-          />
+          <div data-fs-product-details-values-wrapper>
```

Comment:
This can change the CSS cascade and affect styles from existing stores; nothing critical, but it could happen 👀 . @renatamottam WDYT?

Reply:
How are you tracking this kind of modifications side effects?

Reply:
We don't track these modifications since each store can have its customizations and styles. However, each change in our HTML structure can break styles cascading if the existing store implementation covers previous changes, and not current ones yet.